about.md

```diff
@@ -9,16 +9,16 @@ the questions, “Can we use machine learning to create compelling art
 and music? If so, how? If not, why not?”  Our work will be done in
 [TensorFlow](https://www.tensorflow.org), and we’ll release our models and tools in open
 source. These will be accompanied by demos, tutorial
-blog postings and technical papers.  To follow our progress,
+blog postings, and technical papers.  To follow our progress,
 watch our [GitHub](https://github.com/tensorflow/magenta)
 and or join our [discussion group](https://groups.google.com/a/tensorflow.org/forum/#!forum/magenta-discuss).
 
 Magenta encompasses two goals. It’s first a research project to
-advance the state-of-the art in music, video, image and text
```

Comment:
There should be no hyphens. "state-of-the-art" is an adjective. "state of the art" is correct when using as a noun. Both wrong :-)